meshsync/handlers.go

```diff
@@ -241,8 +284,20 @@ func (h *Handler) WatchCRDs() {
 			h.Log.Info("skipping informer resync")
 			return
 		}
+		h.Log.Info("Resyncing informer from watch crd")
 		h.channelPool[channels.ReSync].(channels.ReSyncChannel).ReSyncInformer()
 	}
+
+loop:
+	for {
```

Comment:
do we need a for loop or only a select statement ? ( asking because i didnt find a for loop in the older implentation diff )